ts/components/LabelledItem/index.tsx

```diff
@@ -0,0 +1,163 @@
+/**
+ * Display a labelled, followed by a
+ * input and an icon on the left-end
+ * side of the input
+ *
+ * LABEL
+ * X __________
+ * ^     ^
+ * icon  |
+ *       input
+ */
+import { Item, View } from "native-base";
+import * as React from "react";
+import { useState } from "react";
+import {
+  ImageSourcePropType,
+  ImageStyle,
+  StyleSheet,
+  TextInputProps
+} from "react-native";
+import { TextInputMaskProps } from "react-native-masked-text";
+import { IconProps } from "react-native-vector-icons/Icon";
+import { NavigationEvents } from "react-navigation";
+import I18n from "../../i18n";
+import variables from "../../theme/variables";
+import { WithTestID } from "../../types/WithTestID";
+import { H5 } from "../core/typography/H5";
+import TextInputMask from "../ui/MaskedInput";
+import { Icon } from "./Icon";
+import { Input } from "./Input";
+
+const styles = StyleSheet.create({
+  noBottomLine: {
+    borderBottomWidth: 0
+  },
+  bottomLine: {
+    borderBottomWidth: 1
+  }
+});
+
+// Style type must be generalized so as to support both text icons and image icons
+type StyleType = IconProps["style"] & ImageStyle;
```

Comment:
This type is defined already [in the Icon component](https://github.com/pagopa/io-app/pull/3143/files#diff-9195f2ccdfc413f1e946ecab30d497051cce814f267d4aa729cb923a291b2616R9). 

To reduce duplication, it shall be imported here from there.